src/components/NewTodoForm/NewTodoForm.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { NewTodo } from '../../types/NewTodo';
+
+interface Props {
+  addTodo: (newTodo: NewTodo) => void;
+  users: User[];
+}
+
+export const CreateNewTodo = ({ addTodo, users }: Props) => {
```

Comment:
The name of the component and its file name must be the same. If not, it creates confusion